README.md

```diff
@@ -1,2 +1,40 @@
-# express-typescript-starter
-Starter template for Express.js, Typescript, and the thirdweb TypeScript SDK
+## Using This Template
```

Comment:
can we add a quick title/description of what this template does / is for?

Reply:
yup sure